firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -172,6 +172,9 @@ uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
   int *currentMotorPos;
   int startCol, startRow;
 
+  // Assigned at the end of each movement function to be checked for error handling
```

Comment:
A better way to phrase this would be to say that you're assigning the value using the return from each movement primitive call. Remember, the call executes the function and the result is put in place of the function wherever it was called.